packages/website/plugins/generated-rule-docs.ts

```diff
@@ -22,6 +19,19 @@ const COMPLICATED_RULE_OPTIONS = new Set([
   'member-ordering',
   'naming-convention',
 ]);
+/**
+ * Rules that do funky things with their defaults and require special code
+ * rather than just JSON.stringify-ing their defaults blob
+ */
+const SPECIAL_CASE_DEFAULTS = new Map([
+  //
+  ['ban-types', '[{ /* See below for default options */ }]'],
+]);
```

Comment:
do we want to addres this in separate PR?

Reply:
🤷‍♂️ I can split it out if you think it's worth the effort?
I put it in here cos I was working on the docs as a whole here and leaving this as `const defaultOptions: Options = [{}];` seemed wrong.